src/main/java/games/strategy/engine/framework/map/download/DownloadUtils.java

```diff
@@ -102,6 +103,7 @@ private DownloadUtils() {}
   private static CloseableHttpClient newHttpClient() {
     return HttpClients.custom()
         .setDefaultRequestConfig(RequestConfig.custom().setCookieSpec(CookieSpecs.IGNORE_COOKIES).build())
+        .setRedirectStrategy(new LaxRedirectStrategy())
```

Comment:
Sorry, I should have been clearer in my previous comment.  The line configuring `DefaultRedirectStrategy` is not necessary because `HttpClientBuilder` will use a `DefaultRedirectStrategy` if a custom strategy is not specified.  NBD; functionality is the same. :smile: